src/index.html

```diff
@@ -140,7 +148,51 @@ <h2 class="modal__title">New Post</h2>
       </div>
     </div>
 
-    <script defer src="./scripts/index.js"></script>
-    <script defer src="./scripts/validations.js"></script>
+    <div class="modal" id="avatar-modal">
+      <div class="modal__container">
+        <button type="button" class="modal__close"></button>
+
+        <h2 class="modal__title">Edit profile avatar</h2>
+        <form
+          name="edit-profile-form"
+          id="edit-avatar-form"
+          class="modal__form"
+          novalidate
+        >
+          <label for="profile-avatar-input" class="modal__label"
+            >Profile Picture
+            <input
```

Comment:
* Please, add `type="url"` for the `avatar` input, to validate it as a `link` https://snipboard.io/SNjhbe.jpg